cg/cli/set/families.py

```diff
@@ -0,0 +1,77 @@
+import click
+from .family import family
+from cg.constants import FAMILY_ACTIONS, PRIORITY_OPTIONS
+from cg.store import models, Store
+
+CONFIRM = "Continue?"
+
+
+def _get_samples_by_identifiers(
+    identifiers: click.Tuple([str, str]), store: Store
+) -> [models.Sample]:
+    """Get samples matched by given set of identifiers"""
+    identifier_args = {}
+    for identifier_name, identifier_value in identifiers:
+        identifier_args[identifier_name] = identifier_value
```

Comment:
```suggestion
    identifier_args = dict(identifiers)
```
This will give the same result [ref](https://www.kite.com/python/answers/how-to-convert-a-tuple-to-a-dictionary-in-python)

Reply:
awesome :)